google/cloud/security/iam/inventory/__init__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 Google Inc.
```

Comment:
The copyright header has changed recently, to reflect it is shared with other contributors.  Can you please update this?
```# Copyright 2017 The Forseti Security Authors. All rights reserved.```